assets/bower.json

```diff
@@ -92,6 +97,14 @@
         "src-min-noconflict/theme-dreamweaver.js",
         "src-min-noconflict/theme-eclipse.js"
       ]
+    },
+    "uri.js" : {
+      "main" : [
```

Comment:
we had said we werent going to use main to effectively alter what we get from wiredep, that we were only going to use the exclude in gruntfile for that from now on